pkg/domain/entities/engine_container.go

```diff
@@ -20,7 +20,7 @@ type ContainerEngine interface {
 	ContainerCheckpoint(ctx context.Context, namesOrIds []string, options CheckpointOptions) ([]*CheckpointReport, error)
 	ContainerCleanup(ctx context.Context, namesOrIds []string, options ContainerCleanupOptions) ([]*ContainerCleanupReport, error)
 	ContainerCommit(ctx context.Context, nameOrID string, options CommitOptions) (*CommitReport, error)
-	ContainerCopyFromArchive(ctx context.Context, nameOrID string, path string, reader io.Reader) (ContainerCopyFunc, error)
+	ContainerCopyFromArchive(ctx context.Context, nameOrID, path string, chown bool, reader io.Reader) (ContainerCopyFunc, error)
```

Comment:
Is it OK to change this API? If not I could sneak in the boolean in context.

Reply:
I think it is better to change the API, but lets do it with a struct, so we can add fields in the future.